src/components/Task/Drawer/Form/Form.tsx

```diff
@@ -550,6 +640,32 @@ const TaskDrawerForm = ({
                 </Button>
               </Grid>
             </Grid>
+            <Dialog
+              disableBackdropClick
+              disableEscapeKeyDown
```

Comment:
Is there a good reason for these? They always seem rude to disable on a user.

Reply:
Yeah, you're probably right. I'll remove these. I had noticed currently on mpdx.org that if I have the confirmation modal open to delete a task, and click outside of the modal it will just delete the task without the user confirming. Not sure if that's intentional or not... but I think that's why I went ahead and prevented that here(except here it just calls ```onClose``` and nothing else).